tests/test_bases.py

```diff
@@ -42,3 +130,36 @@ def test_get_root_lengths(canola_h5):
     np.testing.assert_array_almost_equal(
         root_lengths, [20.129579, 62.782368, 80.268003, 34.925591, 3.89724]
     )
+
+
+def test_get_root_lengths_one_point(pts_no_bases):
```

Comment:
I think this one is supposed to be named `test_get_root_lengths_no_roots` and the test below this one should be deleted?

Reply:
Done.